### PR TITLE
Create new events template

### PR DESCRIPTION
**Pull request by AndriyZhuk**
Edit event's-list template and calendar component

### CODE REVIEW

File: client/components/event/calendar/calendar.service.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  // initialize service value
+  this.values = [] ;
+  this.values[0] = new Date();
+  this.values[1] = new Date();
+  this.values[1].setDate(this.values[0].getDate() + 60);
+
+  this.names = [];
+  this.events = [];
+
+  this.click ;
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.service.js]: Remove `this.click`

File: client/components/event/calendar/calendar.service.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  this.clear = function() {
+    this.names = [];
+    this.events = [];
+
+    return;
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.service.js:17]: Remove `return;`

File: client/components/event/calendar/calendar.service.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  this.find_date = function(dateFind, sort) {
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.service.js]: Rename to `findDate`

File: client/components/event/calendar/calendar.service.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    return this.events.filter(function(itEvent) {
+      var temp_date = new Date(itEvent.date_start);
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.service.js]: Use camelCase variables, rename to `tempDate`

File: client/components/event/calendar/calendar.service.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      if (sort == 1) return temp_date > dateFind;
+      return temp_date.toDateString() == dateFind.toDateString();
+    });
+  };
+
+  return {
+    th: this
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.service.js]: Remove `return {th: this}`. Replace with `return calendar;`

File: client/components/event/calendar/calendar.template.html [hunk 6]
```diff
@@ -1,45 +1,41 @@
...
-      <div class="mounth">{{$ctrl.calendar_show_date | date:'MMMM yyyy'}}</div>
-      <div class="glyphicon glyphicon-triangle-right" ng-click="$ctrl.chenge_mounth(1)"></div>
-      <div class="glyphicon glyphicon glyphicon-forward" ng-click="$ctrl.chenge_mounth(12)"></div>
-    </div>
-    <div class="bs-date-header">
-      <div class="bs-day" ng-repeat="dan in $ctrl.calendar_month | limitTo:7 ">{{dan.date | date:'EEE'}}
-      </div>
+    <div class="bs-date row">
+        <div class="bs-date-header btn-group mounth-line row" role="group">
+            <button class="glyphicon glyphicon-backward  btn-default btn-xs" ng-click="$ctrl.chenge_mounth(-12)"></button>
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: Fix variable `$ctrl.chenge_mounth` -> `$ctrl.changeMonth`

File: client/components/event/calendar/calendar.template.html [hunk 7]
```diff
@@ -1,45 +1,41 @@
...
+            <button class="glyphicon glyphicon-triangle-left  btn-default btn-xs" ng-click="$ctrl.chenge_mounth(-1)"></button>
+            <input type="text" class="mounth" disabled value="{{$ctrl.calendar_show_date | date:'MMMM yyyy'}}" />
+            <button class="glyphicon glyphicon-triangle-right  btn-default btn-xs" ng-click="$ctrl.chenge_mounth(1)"></button>
+            <button class="glyphicon glyphicon glyphicon-forward  btn-default btn-xs" ng-click="$ctrl.chenge_mounth(12)"></button>
+        </div>
+        <div class="bs-date-header">
+            <div class="bs-day" ng-repeat="dan in $ctrl.calendar_month | limitTo:7 ">{{dan.date
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: `dan` -> `event`

File: client/components/event/calendar/calendar.template.html [hunk 8]
```diff
@@ -1,45 +1,41 @@
...
+                | date:'EEE'}}
+            </div>
+        </div>
+        <div class="row"></div>
+        <div class="bs-day bs-active{{dan.active}}" ng-repeat="dan in $ctrl.calendar_month" ng-click="$ctrl.carent_day(dan.date)">
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: `carent_day` -> `currentDay`

File: client/components/event/calendar/calendar.template.html [hunk 9]
```diff
@@ -1,45 +1,41 @@
...
+            <button class="btn-danger btn btn-xs"  ng-if="dan.events.length" ng-mousedown="$ctrl.show=dan.events">
+                {{dan.date | date:'dd'}}
+            </button>
+            <button ng-if="!dan.events.length" class="btn btn-default btn-xs">
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: What is `dan.events`?

**Comment by max-yushchuk** [comment]
Fix problem with one event (it is not loading)
Fix problem with links, such as map on event.

File: client/components/event/calendar/calendar.template.html [hunk 10]
```diff
@@ -1,45 +1,41 @@
...
+            'active' for activeDt
+            on dt" ng-click="select(dt.date)" ng-disabled="::dt.disabled" tabindex="-1">
+            <span ng-class="::{'text-muted': dt.secondary, 'text-info': dt.current}" class="ng-binding">01</span></button-->
+                {{dan.date | date:'dd'}}
+            </button>
+        </div>
     </div>
+    <!--button class="btn-warning btn btn-xs"><a href="#!/events/"><span class="menu_l">Go to all events</span></a></button-->
+    <button class="btn-warning btn btn-xs btn-event-link btn-event-link" ng-repeat="dan in $ctrl.show">
+        <a href="#!/events/{{dan.id}}/{{dan.date_start}}"><span class="menu_l">{{dan.name}}</span></a>
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: Refactor to `#!/events/{{event._id}}`

File: client/components/event/calendar/calendar.template.html [hunk 11]
```diff
@@ -1,45 +1,41 @@
...
-      </span>
-      <!--p ng-if="dan.events.length"> >>> {{dan.events[0].name}} <<<</p>
-            <!--div class="bs-date-header">
-                <div class="bs-day" ng-repeat="event in dan.events">{{event.name}}
 
 
-                </div>
-            </div-->
+        </ul>
+        <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: Remove this div

File: client/components/event/event-list/event-list.template.html [hunk 12]
```diff
@@ -1,81 +1,95 @@
-<div class="col-md-9">
-  <div ng-if="give_event.find_date(give_event.date_current).length>0" class="panel panel-success ">
-    <div class="panel-heading">
-      <h3 class="panel-title">Events of selected days</h3>
-    </div>
-  </div>
-
-  <div class="row">
+<span class="event-poster">
+<div class="container h20px">
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html:2]: Remove `h20px`

File: client/components/event/event-list/event-list.template.html [hunk 13]
```diff
@@ -1,81 +1,95 @@
...
+    <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html:3]: Remove empty row

File: client/components/event/event-list/event-list.template.html [hunk 14]
```diff
@@ -1,81 +1,95 @@
...
 
-    <div class="col-md-3" ng-repeat="dan in give_event.find_date(give_event.date_current) | limitTo:4 ">
-      <a href="#!/events/{{dan.id}}/{{dan.date_start}}">
-        <div class="event_block">
-          <img src="assets{{dan.attachments.photo[0]}}" alt="{{dan.name}}" title="{{dan.name}}">
+    <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12 nopadding">
+        <div class="input-group">
+            <input type="text" class="form-control" placeholder="Search for...">
+              <span class="input-group-btn">
+                <button class="btn btn-default" type="button" ng-click="$ctrl.hi()">Go!</button>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Remove `$ctrl.hi()`

File: client/components/event/event-list/event-list.template.html [hunk 15]
```diff
@@ -1,81 +1,95 @@
...
+            <div class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                 aria-expanded="false">
+                {{calendars.names[1]}} : {{calendars.values[1] | date:'dd MMMM yyyy'}}
+                <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
             </div>
-          </div>
+            <ul class="dropdown-menu right-calendar">
+                <calendar></calendar>
+            </ul>
+            <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Remove empty row

File: client/components/event/event-list/event-list.template.html [hunk 16]
```diff
@@ -1,81 +1,95 @@
...
 </div>
-<div class="row">
-  <div class="row">
+<div class="row"></div>
+<div class="container">
+
     <div class="row">
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12 show-event"
+             ng-repeat="dan in dann | orderBy: sort_by | limitTo:howMuch ">
+            <div style="display: none">{{give_event.find_date(give_event.date_current,1)}}</div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Move `display: none` to less

File: client/components/event/calendar/calendar.template.html (hunk 10 above)
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html]: User `ng-href`

File: client/components/event/event-list/event-list.template.html [hunk 17]
```diff
@@ -1,81 +1,95 @@
...
+            <a href="#!/events/{{dan.id}}/{{dan.date_start}}">
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Use `ng-href`
Remove `{{dan.date_start}}`

File: client/components/event/event-list/event-list.template.html [hunk 18]
```diff
@@ -1,81 +1,95 @@
...
+                <div class="text">
+                    <div class="event-title">{{dan.name}}</div>
+                    <div class="event-description">{{dan.description}}</div>
+                    <a href="#!/events/{{dan.id}}/{{dan.date_start}}"></a>
+                    <div class="event-map">
+                        <a href="#!/eventsmap/{{dan.event_points[0].lat}}/{{dan.event_points[0].lon}}/{{dan.type}}/{{dan.date_start}}">
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Refactor to `#!/eventmap`
Use **Map** page

File: client/components/event/event-list/event-list.template.html [hunk 19]
```diff
@@ -1,81 +1,95 @@
...
+                            <img src="assets{{'/img/events/map.png'}}" title="{{dan.name}}">
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Use `ng-src`

File: client/components/event/event-list/event-list.template.html [hunk 20]
```diff
@@ -1,81 +1,95 @@
...
+                        <div class="event-fill">Author:</div>
+                        <div class="event-fill-text">{{dan.owner}}</div>
+                        <div class="row"></div>
+                        <div class="event-fill">Reting:</div>
+                        <div class="event-fill-text">{{dan.rate}}</div>
+                    </div>
+                </div>
+                <div class="fon" style="background-image: url('assets{{dan.photo[0]}}')" alt="{{dan.name}}" title="{{dan.name}}">
+                    <div class="top-title">{{dan.name}}</div>
+                    <!--div class="location">{{$ctrl.event_place(dan.event_points[0].lat,dan.event_points[0].lon)}}</div-->
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Remove comment

File: client/components/event/event-list/event-list.template.html [hunk 21]
```diff
@@ -1,81 +1,95 @@
...
+                </div>
+            </a>
+        </div>
+        <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Remove empty row

File: client/components/event/event-list/event-list.template.html [hunk 22]
```diff
@@ -1,81 +1,95 @@
...
+
     </div>
-  </div>
+
+    <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html]: Remove empty **rows**

File: client/components/event/event-list/events.component.js [hunk 23]
```diff
@@ -1,22 +1,18 @@
...
-      $scope.give_event = eventListService.th;
-      this.danni = $scope.give_event.event;
-    }]
-  });
+    .component('eventList', {
+        templateUrl: 'components/event/event-list/event-list.template.html',
+        controller: ['$scope', '$http', 'calendarService', 'Place', 'Event', function ($scope, $http, calendarService, Place, Event) {
+            $scope.calendars = calendarService.th;
+            $scope.calendars.click = function () {
+                console.log(' Wau !!! ');
```
Comment (yurykozhenov) [inline client/components/event/event-list/events.component.js]: Remove log

File: client/components/event/event-list/events.component.js [hunk 24]
```diff
@@ -1,22 +1,18 @@
...
+                Event.getList().then(function (result) {
+                    $scope.calendars.events = result.concat(event);
+                    $scope.dann = $scope.calendars.events.filter(function (eve) {
+                        return new Date(eve.date_start) >= $scope.calendars.values[0] && new Date(eve.date_end) <= $scope.calendars.values[1];
+                    });
+                    console.log($scope.dan);
```
Comment (yurykozhenov) [inline client/components/event/event-list/events.component.js]: Remove log

File: client/components/event/event-list/events.component.js [hunk 25]
```diff
@@ -1,22 +1,18 @@
...
-        return 'error';
-      });
-
-      $scope.give_event = eventListService.th;
-      this.danni = $scope.give_event.event;
-    }]
-  });
+    .component('eventList', {
+        templateUrl: 'components/event/event-list/event-list.template.html',
+        controller: ['$scope', '$http', 'calendarService', 'Place', 'Event', function ($scope, $http, calendarService, Place, Event) {
```
Comment (yurykozhenov) [inline client/components/event/event-list/events.component.js]: Use `this` instead of `$scope`

Comment (yurykozhenov) [inline client/components/event/event-list/events.component.js]: Remove `$http`

File: client/components/event/event.module.js [hunk 26]
```diff
@@ -1,5 +1,6 @@
 angular.module('event', [
   'eventModel',
+  'placeModel',
```
Comment (yurykozhenov) [inline client/components/event/event.module.js:3]: Remove `placeModel`

File: client/components/place/mapMarkingTypes.factory.js [hunk 27]
```diff
@@ -42,5 +42,21 @@ angular.module('greenTourism')
         color: '#0aff0a'
       }
     };
+
+    types.events = {
+      bicycle: {
```
Comment (yurykozhenov) [inline client/components/place/mapMarkingTypes.factory.js]: Rename keys to `game`, `festival`, etc.

File: server/app/models/event.js [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+  // TODO: add required for almost all fields
+  name: String,
+  description: String,
+  photos: [String],
+  type: String,
+  owner: {
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  date_start: Number,
```
Comment (yurykozhenov) [inline server/app/models/event.js:16]: Rename to camelCase

File: server/app/models/event.js [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+  date_end: Number,
+  location: LocationSchema,
+  price:Number,
```
Comment (yurykozhenov) [inline server/app/models/event.js:19]: Add space after colon

File: server/app/models/event.js (hunk 28 above)
Comment (yurykozhenov) [inline server/app/models/event.js:16]: Change type to `Date`

File: client/app.routes.js [hunk 30]
```diff
@@ -47,11 +47,11 @@ angular.module('greenTourism').config(['$routeProvider', '$locationProvider',
...
         template: '<events-map></events-map>'
       })
-      .when('/eventsmap/:lat/:lng/:type/:date_start', {  // TODO: Remove this from routes (use query strings instead)
-        template: '{{$ctrl.lat}}{{$ctrl.lng}}{{$ctrl.type}}{{$ctrl.date_start}}<events-map></events-map>'  // TODO: Leave only <events-map> here
+      .when('/eventsmap/:eventId', {  // TODO: Remove this from routes (use query strings instead)
+        template: '{{$ctrl.eventId}}<events-map></events-map>'  // TODO: Leave only <events-map> here
       })
-      .when('/events/:eventId/:dataId', { // TODO: Remove this from routes (use query strings instead)
-        template: '{{$ctrl.eventId}}{{$ctrl.dataId}}<event-detail></event-detail>'
+      .when('/events/:eventId', { // TODO: Remove this from routes (use query strings instead)
```
Comment (yurykozhenov) [inline client/app.routes.js]: Remove duplicated `/events/:eventId` route

File: client/app.routes.js [hunk 31]
```diff
@@ -47,11 +47,11 @@ angular.module('greenTourism').config(['$routeProvider', '$locationProvider',
...
+        template: '{{$ctrl.eventId}}<event-detail></event-detail>'
```
Comment (yurykozhenov) [inline client/app.routes.js]: Remove `{{$ctrl.eventId}}`

File: client/app.routes.js [hunk 32]
```diff
@@ -47,11 +47,11 @@ angular.module('greenTourism').config(['$routeProvider', '$locationProvider',
       .when('/eventsmap', {
         template: '<events-map></events-map>'
       })
-      .when('/eventsmap/:lat/:lng/:type/:date_start', {  // TODO: Remove this from routes (use query strings instead)
-        template: '{{$ctrl.lat}}{{$ctrl.lng}}{{$ctrl.type}}{{$ctrl.date_start}}<events-map></events-map>'  // TODO: Leave only <events-map> here
+      .when('/eventsmap/:eventId', {  // TODO: Remove this from routes (use query strings instead)
+        template: '{{$ctrl.eventId}}<events-map></events-map>'  // TODO: Leave only <events-map> here
```
Comment (yurykozhenov) [inline client/app.routes.js:51]: Remove `{{$ctrl.eventId}}`

File: client/components/event/calendar/calendar.component.js [hunk 33]
```diff
@@ -2,25 +2,31 @@ angular.module('calendar', []);
 
 angular.module('calendar').component('calendar', {
   templateUrl: 'components/event/calendar/calendar.template.html',
-  controller: ['$http', '$scope', 'eventListService', function($http, $scope, eventListService) {
-    $scope.give_event = eventListService.th;
+  controller: ["calendarService", "$scope", function(calendarService, $scope) {
 
-    this.calendar_carent_date = new Date();
+    $scope.service = calendarService;
+    console.log('service.names.length = '+($scope.service.names.length)) ;
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.component.js]: Remove `console.log`

File: client/components/event/calendar/calendar.component.js [hunk 34]
```diff
@@ -2,25 +2,31 @@ angular.module('calendar', []);
...
+      this.calendarShowDate = $scope.service.values[0];
+    }
+    if ($scope.service.names.length==1) {
+      this.name = 'To' ;
+      this.calendarShowDate = $scope.service.values[1] ;
+    }
 
-    this.name = $scope.give_event.CalendarName.length;
+      $scope.service.names.push(this.name) ;
+    console.log(' this.name= ' + this.name + ' service.names['+($scope.service.names.length-1)+']= ' + $scope.service.names[$scope.service.names.length-1]) ;
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.component.js]: Remove `console.log`

File: client/components/event/calendar/calendar.component.js [hunk 35]
```diff
@@ -32,47 +38,44 @@ angular.module('calendar').component('calendar', {
         return date.getDate();
       }
 
-      var start_mounth = getFirstWeekDayOfMonth(some_date.getFullYear(), (some_date.getMonth()));
-      if (start_mounth == 0) start_mounth = 7;
-      var end_mounth = getLastDateOfMonth(some_date.getFullYear(), (some_date.getMonth()));
+      var startMounth = getFirstWeekDayOfMonth(someDate.getFullYear(), (someDate.getMonth()));
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.component.js]: Type: `startMount` -> `startMonth`

File: client/components/event/calendar/calendar.component.js [hunk 36]
```diff
@@ -32,47 +38,44 @@ angular.module('calendar').component('calendar', {
...
     };
 
-    this.chenge_mounth = function(step) {
-      var it = this.calendar_show_date.getMonth();
-      this.calendar_show_date.setMonth(it + step);
-      this.calendar_init(this.calendar_show_date);
-
-      $scope.give_event.date_current = this.calendar_show_date;
+    this.changeMonth = function(step) {
+      var it = this.calendarShowDate.getMonth();
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.component.js]: `it` -> `month`

File: client/components/event/calendar/calendar.template.html [hunk 37]
```diff
@@ -1,45 +1,32 @@
...
+            <button ng-if="!dateCalendar.events.length" class="btn btn-default btn-xs">
+                {{dateCalendar.date | date:'dd'}}
+            </button>
+        </div>
     </div>
-    <div class="bs-date-header">
-      <div class="bs-day" ng-repeat="dan in $ctrl.calendar_month | limitTo:7 ">{{dan.date | date:'EEE'}}
-      </div>
+    <!--button class="btn-warning btn btn-xs"><a href="#!/events/"><span class="menu_l">Go to all events</span></a></button-->
+    <button class="btn-warning btn btn-xs btn-event-link btn-event-link" ng-repeat="dateCalendar in $ctrl.show">
```
Comment (yurykozhenov) [inline client/components/event/calendar/calendar.template.html:27]: Rename `dateCalendar` -> `event`

File: client/components/event/event-detail/event-detail.template.html [hunk 38]
```diff
@@ -1,19 +1,19 @@
...
+        <li class="media" ng-repeat="img in event.photo">
           <div class="media-middle">
             <img src="assets{{img}}" ng-class="$first ? 'img-responsive' : 'media-object center-block img-small'"
                 alt="{{event.name}}">
           </div>
           <div ng-if="$first" class="media-body">
             <h4 class="text-center center-block glyphicon glyphicon-heart" style="color: red">
               {{event.event_rate}}</h4>
-            <a href="#!/eventsmap/{{event.event_points[0].lat}}/{{event.event_points[0].lon}}/{{event.type}}/{{event.date_start}}">
+            <a ng-href="#!/places/">
```
Comment (yurykozhenov) [inline client/components/event/event-detail/event-detail.template.html]: `ng-href` is unnecessary here

File: client/components/event/event-detail/event-list.js [hunk 39]
```diff
@@ -7,34 +7,34 @@ angular.module('eventDetail').service('eventListService', ['$http', function($ht
...
-      if (sort == 1) return temp_date > dateFind;
-      return temp_date.toDateString() == dateFind.toDateString();
+      var tempDate = new Date(itEvent.date_start);
+      if (sort == 1) return tempDate > dateFind;
+      return tempDate.toDateString() == dateFind.toDateString();
     });
   };
 
-  this.find_distanse = function(lat, lng, event) {
+  this.findDistanse = function(lat, lng, event) {
```
Comment (yurykozhenov) [inline client/components/event/event-detail/event-list.js]: Typo: `findDistanse` -> `findDistance`

File: client/components/event/event-detail/event-list.js [hunk 40]
```diff
@@ -7,34 +7,34 @@ angular.module('eventDetail').service('eventListService', ['$http', function($ht
...
     function compareEvent(eventA, eventB) {
-      var comperA = +Math.pow(Math.abs(+eventA.event_points[0].lat - lat), 2) + Math.pow(Math.abs(+eventA.event_points[0].lon - lng), 2);
-      var comperB = +Math.pow(Math.abs(+eventB.event_points[0].lat - lat), 2) + Math.pow(Math.abs(+eventB.event_points[0].lon - lng), 2);
+      var comperA = +Math.pow(Math.abs(+eventA.eventPoints[0].lat - lat), 2) + Math.pow(Math.abs(+eventA.eventPoints[0].lon - lng), 2);
```
Comment (yurykozhenov) [inline client/components/event/event-detail/event-list.js:25]: Remove unnecessary `+` before `Math.pow`

Comment (yurykozhenov) [inline client/components/event/event-detail/event-list.js:25]: Typo: `comper` -> `compare`

File: client/components/event/event-list/event-list.template.html [hunk 41]
```diff
@@ -1,81 +1,83 @@
...
+            <div class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                 aria-expanded="false">
+                {{calendars.names[0]}} : {{calendars.values[0] | date:'dd MMMM yyyy'}}
+                <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
             </div>
-          </div>
+            <ul class="dropdown-menu">
+                <calendar></calendar>
+            </ul>
+            <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html:24]: Remove empty row

File: client/components/event/event-list/event-list.template.html [hunk 42]
```diff
@@ -1,81 +1,83 @@
...
+                        <div class="event-fill-text">{{event.price}}</div>
+                        <div class="row"></div>
+                        <div class="event-fill">Author:</div>
+                        <div class="event-fill-text">{{event.owner}}</div>
+                        <div class="row"></div>
+                        <div class="event-fill">Reting:</div>
+                        <div class="event-fill-text">{{event.rate}}</div>
+                    </div>
+                </div>
+                <div class="fon" style="background-image: url('assets{{event.photo[0]}}')" alt="{{event.name}}" title="{{event.name}}">
```
Comment (yurykozhenov) [inline client/components/event/event-list/event-list.template.html:74]: User `ng-style`

File: client/components/event/event-list/events.component.js [hunk 43]
```diff
@@ -1,22 +1,32 @@
...
+    })
+  };
 
-      $scope.give_event = eventListService.th;
-      this.danni = $scope.give_event.event;
-    }]
-  });
+  this.findWord = function(reg){
+    $scope.events = $scope.calendars.events.filter(function(event) {
+      console.log('event.name'+event.name+' reg= ' + reg) ;
```
Comment (yurykozhenov) [inline client/components/event/event-list/events.component.js]: Remove `console.log`

File: client/components/event/events-map/event-map.component.js [hunk 44]
```diff
@@ -81,14 +85,14 @@ angular.module('eventsMap', ['calendar', 'eventMapType'])
...
-          $scope.filters = $scope.eventL.find_event($scope.eventL.CalendarName[0].calendar_show_date, $scope.eventL.CalendarName[1].calendar_show_date, $scope.eventL.active_type());
+          $scope.filters = $scope.eventL.find_event($scope.eventL.CalendarName[0].calendarShowDate, $scope.eventL.CalendarName[1].calendarShowDate, $scope.eventL.active_type());
           var clicks_arr = [];
           var sort_arr = [];
 
           for (var i = 0; i < $scope.filters.length; i++) {
             if (($scope.filters[i].event_points[0].lat == lat) && ($scope.filters[i].event_points[0].lon == lng)) {
               clicks_arr.push($scope.filters[i]); //alert($scope.filters[i].name);
-              sort_arr.push($scope.filters[i].date_start - $scope.eventL.CalendarName[0].calendar_show_date);
+              sort_arr.push($scope.filters[i].date_start - $scope.eventL.CalendarName[0].calendarShowDate);
```
Comment (yurykozhenov) [inline client/components/event/events-map/event-map.component.js:95]: Rename `sort_arr` -> `sortArr`

File: client/components/event/events-map/event-map.component.js [hunk 45]
```diff
@@ -97,11 +101,11 @@ angular.module('eventsMap', ['calendar', 'eventMapType'])
         };
 
         this.temp_click = function() {
-          if ($scope.eventL.active_type().length != 0) $scope.filters = $scope.eventL.find_event($scope.eventL.CalendarName[0].calendar_show_date, $scope.eventL.CalendarName[1].calendar_show_date, $scope.eventL.active_type());
+          if ($scope.eventL.active_type().length != 0) $scope.filters = $scope.eventL.find_event($scope.eventL.CalendarName[0].calendarShowDate, $scope.eventL.CalendarName[1].calendarShowDate, $scope.eventL.active_type());
```
Comment (yurykozhenov) [inline client/components/event/events-map/event-map.component.js:104]: Rename `find_event` -> `findEvent`

File: client/components/event/events-map/event-map.template.html [hunk 46]
```diff
@@ -24,29 +24,33 @@
...
+            <div class="btn-group">
+              <div class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                   aria-expanded="false">
+                {{calendars.names[0]}} : {{calendars.values[0] | date:'dd MMMM yyyy'}}
+                <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
+              </div>
+              <ul class="dropdown-menu">
+                <calendar></calendar>
+              </ul>
+              <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/events-map/event-map.template.html]: Remove empty row

File: client/components/event/events-map/event-map.template.html [hunk 47]
```diff
@@ -24,29 +24,33 @@
...
+            <div class="btn-group">
+              <div class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                   aria-expanded="false">
+                {{calendars.names[1]}} : {{calendars.values[1] | date:'dd MMMM yyyy'}}
+                <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
+              </div>
+              <ul class="dropdown-menu right-calendar">
+                <calendar></calendar>
+              </ul>
+              <div class="row"></div>
```
Comment (yurykozhenov) [inline client/components/event/events-map/event-map.template.html]: Remove empty row

File: client/components/event/events-map/events-map-type/event-map-type.component.js [hunk 48]
```diff
@@ -7,7 +7,7 @@ angular.module('eventMapType', ['event'])
       this.data = $http.get('components/event/event-list/event.data.json').success(function(data) {
```
Comment (yurykozhenov) [inline client/components/event/events-map/events-map-type/event-map-type.component.js:7]: Remove `$http`

File: client/assets/less/pages/events.less [hunk 49]
```diff
@@ -88,12 +88,192 @@
...
+  position: relative;
+}
+.show-event:hover .text{
+  display: block;
+  position: absolute;
+}
+.h20px {
+  margin: 20px auto;
+}
+.show-event A {
```
Comment (yurykozhenov) [inline client/assets/less/pages/events.less:221]: Use lowercase tag names `A` -> `a`

**Pull request closed without merging**